src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.spec.ts

```diff
@@ -53,13 +58,16 @@ describe('ConstraintsPanelComponent', () => {
         // Estimated cost in $ per acre
         estimatedCost: ['', [Validators.min(0), Validators.required]],
         // Max cost of treatment for entire planning area
-        maxCost: ['', Validators.min(0)],
+        maxCost: [{ value: '', disabled: true }, Validators.min(0)],
       }),
       physicalConstraintForm: fb.group({
         // Maximum slope allowed for planning area
         maxSlope: ['', Validators.min(0)],
-        // Maximum road distance
-        maxRoadDistance: ['', Validators.min(0)],
+        // Minimum distance from road allowed for planning area
+        // TODO: Update variable name to minDistanceFromRoad
+        minDistanceFromRoad: ['', Validators.min(0)],
+        // Maximum area to be treated in acres 
+        maxArea: ['', [Validators.min(0), Validators.required]],
         standSize: ['Large', Validators.required],
```

Comment:
Similar to my previous comment, `standSize` is not actually defined on the constraints panel but on the parent component (createScenario) so this is coupling both components.

Is it possible to move the constraints partially declared on `createScenario` to this component? That way we'll have the whole definition on one place, and we can use the components independently.

Reply:
Similar to previous discussion — will add a TODO!